marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java

```diff
@@ -58,18 +58,15 @@ public SaveReceipt updateResource(String payload, String resourceId) {
             return receipt;
         }
 
-        private boolean cannotUpdateAmps(){
-            try{
-                Versions.MarkLogicVersion mlVersion = new Versions().getMLVersion(getManageClient().getXml("/manage")
-                    .getElementValue("/node()/c:version"));
-                // Users should be able to update amps on versions > 10.0-4.4
-                return mlVersion.isNightly() == true ? false: (mlVersion.getMajor() == 10 && mlVersion.getMinor() == 404);
+        private boolean cannotUpdateAmps() {
+            try {
+                return new MarkLogicVersion(getManageClient().getXml("/manage")
+                        .getElementValue("/node()/c:version")).cannotUpdateAmps();
```

Comment:
I like this "cannotUpdateAmps" method - this is a good example of the benefit of having a class here, particularly in that it consolidates all of the logic that we have for checking the version number. 

Pinging @anu3990  and @SameeraPriyathamTadikonda  and @srinathgit  about this - we'll soon add a method to the MarkLogicVersion class to determine if Optic rowIDs are supported - i.e. if ML is 10.0-5 or higher. 

Which means MarkLogicVersion will likely make its way into the API project soon (that's out of scope for this ticket).